skimage/measure/profile.py

```diff
@@ -56,6 +61,22 @@ def profile_line(image, src, dst, linewidth=1,
     array([ 1.,  1.,  1.,  2.,  2.,  2.,  0.])
     >>> profile_line(img, (1, 0), (1, 5))  # This accesses the full first row
     array([ 1.,  1.,  1.,  2.,  2.,  2.])
+
+    For different reduce_func inputs:
+    >>> profile_line(img, (1, 0), (1, 3), linewidth=3, reduce_func=np.mean)
+    array([ 0.66666667,  0.66666667,  0.66666667,  1.33333333])
+    >>> profile_line(img, (1, 0), (1, 3), linewidth=3, reduce_func=np.max)
+    array([1, 1, 1, 2])
+    >>> profile_line(img, (1, 0), (1, 3), linewidth=3, reduce_func=np.sum)
+    array([2, 2, 2, 4])
+
+    The full array will be returned when reduce_func=None.
+    >>> profile_line(img, (1, 2), (4, 2), linewidth=3, order=0,
```

Comment:
I think adding a blank line above this one will fix the CI errors. Without it the parser doesn't recognize this as a code block and is confused by the indent below. Same applies to the paragraph above (line 66).